src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add(digit) {
+      return (this.result += digit);
+    },
+    subtract(digit) {
+      return (this.result -= digit);
+    },
+    multiply(digit) {
+      return (this.result *= digit);
```

Comment:
As with `add` and `subtract`, the `multiply` method should not return the result.